### PR TITLE
Dropped support for iOS 13

### PR DESCRIPTION
**Pull request by ajaysubra**
# Description

This is going out in the next major version and we'll be dropping support for iOS 13 and min deployment version support will be iOS 14.

# Check List

- [ ] Are you changing anything with the public API?
- [ ] Have you tested this change on real device?
- [ ] Are your changes backwards compatible with previous SDK Versions?
- [ ] Have you added unit test coverage for your changes?
- [ ] Have you verified that your changes are compatible with all the operating system version this SDK currently supports?

# Manual Test Plan

1.

# Supporting Materials

### CODE REVIEW

**Review by evan-masseau — APPROVED** [review]

**Comment by evan-masseau** [comment]
If this is standard practice then we may as well do it in a major release. 
I do have slight reservation that we aren't really doing it for a _positive reason_ like to attain new features, which makes it hard to justify to a developer who maybe has an app that still supports older OSes. But perhaps that is a very small/non-existent segment.

**Review by kennyklaviyo — APPROVED** [review]

File: KlaviyoSwift.podspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.source           = { :git => "https://github.com/klaviyo/klaviyo-swift-sdk.git", :tag => s.version.to_s }
   s.swift_version = '5.7'
   s.platform = :ios
-  s.ios.deployment_target = '13.0'
+  s.ios.deployment_target = '14.0'
```
Comment (kennyklaviyo) [inline KlaviyoSwift.podspec:16]: > If this is standard practice then we may as well do it in a major release.
> I do have slight reservation that we aren't really doing it for a positive reason like to attain new features, which makes it hard to justify to a developer who maybe has an app that still supports older OSes. But perhaps that is a very small/non-existent segment.

If this is true, we'd lose 0.7% of devices
https://iosref.com/ios-usage

Comment (kennyklaviyo) [reply KlaviyoSwift.podspec:16]: @evan-masseau

Comment (ajaysubra) [reply KlaviyoSwift.podspec:16]: Had a discussion internally and decided to leave this at 13 for now.

**Comment by ajaysubra** [comment]
Had a discussion internally and decided to leave this at 13 for now.

**Pull request closed without merging**